models/solutions.js

```diff
@@ -102,6 +102,7 @@ module.exports = {
       reportInformation : Object,
       certificateTemplateId : "ObjectId",
       rootOrganisations : Array,
-      createdFor : Array
+      createdFor : Array,
+      projectTemplateId : "ObjectId"
```

Comment:
@VISHNUDAS-tunerlabs this model change required in all service

Reply:
@ankitpws Yes will do that

Reply:
@VISHNUDAS-tunerlabs - We should keep this key as index in all services consistent